Content.Server/Electrocution/ElectrocutionSystem.cs

```diff
@@ -147,6 +148,10 @@ private void OnElectrifiedInteractUsing(EntityUid uid, ElectrifiedComponent elec
             if (!electrified.OnInteractUsing)
                 return;
 
+            // Make an exception for floor tiles, to make placing them around exposed wires less of a hassle.
+            if (HasComp<FloorTileItemComponent>(args.Used))
+                return;
+
```

Comment:
Do you know if disposal pipes are also okay?

Reply:
Both disposal and atmos pipes have to be wrenched into place, so you have to drag/q them to the right spot then click on them with a wrench. So they should be fine.

Because wires can also go on lattice, to which you can add metal sheets, maybe I should also exclude metal sheets? But I don't think that problem is anywhere near as common as people just trying to (re)place tiles on existing powered sections of a station. And if you're building new wiring on lattice you probably have gloves anyways.

Reply:
I will however add the RCD in